hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieIndex.java

```diff
@@ -749,6 +749,67 @@ public void testRecordIndexTagLocationAndUpdate(boolean populateMetaFields) thro
     assertEquals(newInsertsCount, recordLocations.filter(entry -> newPartitionPath.equalsIgnoreCase(entry._1.getPartitionPath())).count());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = "INMEMORY")
```

Comment:
Why parameterizing the test with only one value?

Reply:
Only in_memory and record_index are explicit indexes which have updateLocation() support. For other indexes, there is no handling for deletes as they are implicit with storage and read the current record keys from parquet each time.

Reply:
We have an explicit test for RLI delete in TestHoodieBackedMetadata. So I did not add RECORD_INDEX in the parameter here to reduce test time.

Reply:
Then just use `@Test` should be fine.

Reply:
fixed